nshankaz12/SBPD-Help-Desk
# Finished Assign Role Feature for Admin - AssignRole Page

Completed the functionality where admin can select a user from a drop down, assign that user to a new role and click the Assign Role button. For now the drop down is just user's first names. However, if 2 or more people have the same first name, then there is no way to distinguish them. My suggestion would be to either make first and last name one field or make the drop down field the user's email instead which is always going to be unique.

TO TEST THIS FEATURE FOR REVIEW PURPOSES:
- Go to AssignRole page
- From first drop down, select your name.
- From second drop down, select a role that is not your current role.
      -  If you don't know your current role, go to the Playground page to see it.
- After selecting second drop down, click the red AssignRole button.
- To see if the function actually works, go to Playground page and see if the new role you selected does match what role it says you have.
